### PR TITLE
[Pulsar Manager]Add release notes for puslar manager 0.2.0

### PR DESCRIPTION
**Pull request by tuteng**
<!--
### Contribution Checklist
  
  - Name the pull request in the form "[Issue XYZ][component] Title of the pull request", where *XYZ* should be replaced by the actual issue number.
    Skip *Issue XYZ* if there is no associated github issue for this pull request.
    Skip *component* if you are unsure about which is the best component. E.g. `[docs] Fix typo in produce method`.

  - Fill out the template below to describe the changes contributed by the pull request. That will give reviewers the context they need to do the review.
  
  - Each pull request should address only one issue, not mix up code from multiple issues.
  
  

### Motivation

* Add release notes for puslar manager 0.2.0

### Modifications

* Update release note file
* Update readme for pulsar manager

### Verifying this change

* Local test pass

### CODE REVIEW

File: site2/website/pulsar-manager-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 ## Apache Pulsar Manager
 
+### 0.2.0 &mdash; 2020-09-28 <a id="0.2.0"></a>
+
+#### Fixes
+
+* Support multi address for broker stats [#306](https://github.com/apache/pulsar-manager/pull/306)
+* Use PulsarAdmin instead of HttpUtil [#315](https://github.com/apache/pulsar-manager/pull/315)
+* Serve frontend directly from Pulsar Manager backend process [#288](https://github.com/apache/pulsar-manager/pull/288)
+* Support docker for jwt [#218](https://github.com/apache/pulsar-manager/pull/218)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:10]:
```suggestion
* Support docker for JWT [#218](https://github.com/apache/pulsar-manager/pull/218)
```

File: site2/website/pulsar-manager-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+* Support sub and unsub [#240](https://github.com/apache/pulsar-manager/pull/240)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:11]:
```suggestion
* Support sub and unsub operations [PR-240](https://github.com/apache/pulsar-manager/pull/240).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+* Support peek messages from Ppulsar broker [#241](https://github.com/apache/pulsar-manager/pull/241)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:12]:
```suggestion
* Support peeking messages from the Pulsar broker [PR-241](https://github.com/apache/pulsar-manager/pull/241).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+* Support BbookKeeper visual manager 1.2.0 [#300](https://github.com/apache/pulsar-manager/pull/300)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:13]:
```suggestion
* Support BookKeeper visual manager 1.2.0 [PR-300](https://github.com/apache/pulsar-manager/pull/300).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 5]
```diff
@@ -1,5 +1,25 @@
...
+* Support https forward [#258](https://github.com/apache/pulsar-manager/pull/258)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:14]: Support forwarding messages through HTTPS?

File: site2/website/pulsar-manager-release-notes.md [hunk 6]
```diff
@@ -1,5 +1,25 @@
...
+* Support display stats for tenant and namespace [#243](https://github.com/apache/pulsar-manager/pull/243)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:15]:
```suggestion
* Support displaying stats for tenants and namespaces [PR-243](https://github.com/apache/pulsar-manager/pull/243).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 7]
```diff
@@ -1,5 +1,25 @@
...
+* Add a configuration file for backend service of pulsar manager [#236](https://github.com/apache/pulsar-manager/pull/236)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:16]:
```suggestion
* Add a configuration file for the backend service of Pulsar manager [PR-236](https://github.com/apache/pulsar-manager/pull/236).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 8]
```diff
@@ -1,5 +1,25 @@
...
+* Add default configuration for environment [#242](https://github.com/apache/pulsar-manager/pull/242)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:17]:
```suggestion
* Add default configurations for the environment [PR-242](https://github.com/apache/pulsar-manager/pull/242).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 9]
```diff
@@ -1,5 +1,25 @@
...
+* fixed a SQL syntax error [#298](https://github.com/apache/pulsar-manager/pull/298)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:18]:
```suggestion
* Fixe an SQL syntax error [PR-298](https://github.com/apache/pulsar-manager/pull/298).
```

File: site2/website/pulsar-manager-release-notes.md [hunk 10]
```diff
@@ -1,5 +1,25 @@
...
+* Fix the pulsar-manager can not process the request by pulsar proxy [#281](https://github.com/apache/pulsar-manager/pull/281)
```
Comment (Huanli-Meng) [inline site2/website/pulsar-manager-release-notes.md:19]:
```suggestion
* Fix the issue that Pulsar Manager fail to process the request sent to the Pulsar proxy [PR-281](https://github.com/apache/pulsar-manager/pull/281).
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave the comments. PTAL

File: site2/website/pulsar-manager-release-notes.md [hunk 11]
```diff
@@ -1,19 +1,39 @@
 ## Apache Pulsar Manager
 
+### 0.2.0 &mdash; 2020-09-28 <a id="0.2.0"></a>
+
+#### Fixes
```
Comment (Jennifer88huang) [inline site2/website/pulsar-manager-release-notes.md:5]: I suggest that we just remove this heading.
For guides, refer to [heading usage](https://developers.google.com/style/headings)

File: site2/website/pulsar-manager-release-notes.md [hunk 12]
```diff
@@ -1,19 +1,39 @@
...
+* Add a configuration file for the backend service of Pulsar manager [PR-236](https://github.com/apache/pulsar-manager/pull/236).
+* Add default configurations for the environment [PR-242](https://github.com/apache/pulsar-manager/pull/242).
+* Fixe an SQL syntax error [PR-298](https://github.com/apache/pulsar-manager/pull/298).
+* Fix the issue that Pulsar Manager fail to process the request sent to the Pulsar proxy [PR-281](https://github.com/apache/pulsar-manager/pull/281).
+* Change the default port and replace the request URI [PR-316](https://github.com/apache/pulsar-manager/pull/316).
+
+
 ### 0.1.0 &mdash; 2019-11-25 <a id="0.1.0"></a>
 
 #### Fixes
```
Comment (Jennifer88huang) [inline site2/website/pulsar-manager-release-notes.md:25]: remove the heading.

File: site2/docs/administration-pulsar-manager.md [hunk 13]
```diff
@@ -14,38 +14,38 @@ Pulsar Manager is a web-based GUI management and monitoring tool that helps admi
...
+curl \
+    -H 'X-XSRF-TOKEN: $CSRF_TOKEN' \
+    -H 'Cookie: XSRF-TOKEN=$CSRF_TOKEN;' \
+    -H "Content-Type: application/json" \
+    -X PUT http://localhost:7750/pulsar-manager/users/superuser \
+    -d '{"name": "admin", "password": "apachepulsar", "description": "test", "email": "username@test.org"}'
+```
 
-You can find the in the [Docker](https://github.com/apache/pulsar-manager/tree/master/docker) directory and build an image from scratch as well:
+You can find the in the [Docker](https://github.com/apache/pulsar-manager/tree/master/docker) directory and build an image from source code as well:
```
Comment (Jennifer88huang) [inline site2/docs/administration-pulsar-manager.md:39]: find the xxx? What's the modifier of "the"?

**Review by Jennifer88huang — APPROVED** [review]

**Review by sijie — APPROVED** [review]

**Pull request merged**